packages/rum/src/domain/record/serializationUtils.spec.ts

```diff
@@ -95,3 +101,89 @@ describe('getElementInputValue', () => {
     })
   })
 })
+
+describe('Build stylesheet absolute url', () => {
+  const cssHref = 'https://site.web/app-name/static/assets/resource.min.css'
+  const resolvedPath = 'https://site.web/app-name/static/assets/fonts/fontawesome-webfont.eot'
+
+  describe('convert relative path to absolute', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about also replacing `path` references by `url` in tests?